nab/detectors/spatial/spatial_detector.py

```diff
@@ -0,0 +1,30 @@
+from nab.detectors.base import AnomalyDetector
+
+SPATIAL_TOLERANCE = 0.05
+
+class SpatialDetector(AnomalyDetector):
```

Comment:
please add a short note that this is a simple threshold detector which is also used in NumentaDetector, and that this file is a demonstration how "powerful" it is.

Reply:
Maybe comeup with a different name? This could be confusing for "spatial pooler"...threshold detector, ...?